EEPS/eep-4.md

```diff
@@ -0,0 +1,277 @@
+
+## Specification
+
+### General Behaviour/Definitions (Apply to all proposal types)
+
+Any `vote_value` that is submitted that does not fall into the expected results will be only be used to count
+towards voter participation of the overall vote (if that metric is being observed). For example, for proposal type
+`referendum-v1`, a `vote_value` of `7` shall not count towards `No` or `Yes`, but the staked amount will count towards
+voter participation for the "15% voter participation threshold".
+
+Votes are weighted using information from the `staked` table for voter info. If a user has not voted, then the weight
+of their vote will be taken from the `delband` table. To clarify, a voter's weight should be equal to their self-staked
+and staked-towards-another-account amount. Stake that has been delegated towards an account will not be used when
+calculating the weight of their vote. This is to avoid a user going to a delegation service and renting resources
+to enhance their voting weight.
+
+Proxy voting should be handled by first delegating the `staked` amount of a user towards its proxy. If the user
+casts a vote on a `proposal`, then their `stake` shall be removed from their proxy and weighted towards their 
+chosen `vote_value` on its own. 
+
+### Referendum
+
+`referendum-v1` should be seen as the only `type` that Block Producers should utilize as a signal 
+to act upon. The available responses are always `0`:`No`, `1`:`Yes`
+All UIs should display the voting buttons along with their value e.g. "0 - No" and "1 - Yes"
+so that a user can easily verify their vote through any block explorer, without the need to already
+have the knowledge that a `vote_value` of 0 properly signifies their vote of "No", and so on.
+
+```
+{
+  "type":"referendum-v1",
+  "content":"content here"
+}
+```
+
+Example:
+```
+{
+  "type":"referendum-v1",
+  "content":"Should EOS change its symbol to SYS?"
+}
+```
+**_Summary Notes_**
+
+Standard `vote_value`: 
+
+`0` - "No" 
+
+`1` - "Yes" 
+
+If a vote of `type`:`referendum-v1` passes the required thresholds as defined in the goverenance documents,
+Block Producers should act upon the result of this referendum.
+
+As of writing, Article XI of the Constitution reads as: 
+
+"This Constitution and its subordinate documents shall not be amended except by a vote of the token holders with no less than 15% vote participation among tokens and no fewer than 10% more Yes than No votes, sustained for 30 continuous days within a 120 day period."
+
+_See [pre-launch discussion](https://forums.eosgo.io/discussion/820/article-xvi-v0-3-0-draft-eos-io-constitution-amendment#latest) for further clarification_
+
+"15% vote participation" is defined as `staked`:`total` / `total_EOS_currency_supply` being above 0.15 or not.
+
+"10% more Yes than No votes" is defined as a check on whether `staked`:`1` > ( 0.55 * ( `staked`:`0` + `staked`:`1` )) 
```

Comment:
The formulae is not easy to read, would also probably use full backticks for the formulae instead of parts.